package.json

```diff
@@ -3,11 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@chakra-ui/react": "2.5.4",
```

Comment:
I think moving forward we want to avoid using @chakra-ui/react as much as possible. It has caused some compatibility issues with the web SDK. This might be okay considering that it is a separate example app that just imports the webSDK. Just an fyi when developing on the sdk itself

Reply:
Yeah totally, I would love to get chakra ripped out of the SDK! 

I actually was running into issue using styled components and the modal (chakra from our SDK was injecting some unwanted CSS). We also were using some chakra components here previously (tooltips & buttons) so figured I'd just add it and use it until we do a true redesign of the demo!